unit-tests/src/test/java/org/eclipse/collections/impl/map/mutable/MutableMapIterableTestCase.java

```diff
@@ -608,6 +608,45 @@ public void withMapNull()
         Assert.assertThrows(NullPointerException.class, () -> this.newMap().withMap(null));
     }
 
+    @Test
+    public void putAllMapIterable()
+    {
+        MutableMapIterable<String, Integer> map = this.newMapWithKeysValues("A", 1, "B", 2);
+        MutableMapIterable<String, Integer> simpleMap = Maps.mutable.with("B", 22, "C", 3);
+        map.putAllMapIterable(simpleMap);
+        Verify.assertMapsEqual(UnifiedMap.newWithKeysValues("A", 1, "B", 22, "C", 3), map);
```

Comment:
Can be replaced with `Maps.mutable.with()` instead of `UnifiedMap`

Reply:
Updated to use Maps.mutable.with() instead of UnifiedMap methods. Updated the same for putallmapiterable test cases too.